docs/design/README.md

```diff
@@ -3693,7 +3693,7 @@ tools. However, there is a large gap in programming models between the two
 languages, generally requiring a revision to the architecture. So, thus far the
 common pattern in the Rust community is to "rewrite it in Rust"
 ([1](https://deprogrammaticaipsum.com/the-great-rewriting-in-rust/),
-[2](https://unhandledexpression.com/rust/2017/07/10/why-you-should-actually-rewrite-it-in-rust.html),
+[2](http://unhandledexpression.com:8081/rust/2017/07/10/why-you-should-actually-rewrite-it-in-rust.html),
```

Comment:
insecure, not ideal.

wayback machine could be a better option?

Reply:
Yeah, I'd lean towards wayback, I'm not able to reliably load this link.

```suggestion
[2](https://web.archive.org/web/20230923033736/https://unhandledexpression.com/rust/2017/07/10/why-you-should-actually-rewrite-it-in-rust.html),
```